sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets/README.md

```diff
@@ -0,0 +1,252 @@
+        print(secret.id)
+        print(secret.name)
+```
+
+## Troubleshooting
+### General
+The Key Vault APIs generate exceptions that can fall into one of the azure-core defined exceptions. For more detailed infromation about exceptions and how to deal with them, see [Azure Core exceptions](TODO).
+
+For example, if you try to retrieve a secret after it is deleted a `404` error is returned, indicating resource not found. In the following snippet, the error is handled gracefully by catching the exception and displaying additional information about the error.
+```python
+try:
+    client.get_secret("deleted_secret")
+except ResourceNotFoundError as e:
+    if e.code == 404:
+        print(e.message)
+    else:
+        raise
+    
+Output: "Secret not found:deleted_secret"
+```
+### Logging [TODO]
+This SDK uses Python standard logging library. You can configure logging print out debugging information to the stdout or anywhere you want.
+
+```python 
+import logging
+logging.basicConfig(level=logging.DEBUG)
+```
+Http request and response details are printed to stdout with this logging config.
+
+## Next steps
+Several KeyVault Python SDK samples are available to you in the SDK's GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Key Vault:
+* [test_examples_keyvault.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/keyvault/azure-security-keyvault/tests/test_examples_keyvault.py) - Contains code snippets found in this article.
+* [test_examples_secrets_sync.py](TODO) -  Python code for working with Key Vault secrets, including:
+    * Create a secret
+    * Get an existing secret
+    * Update an existing secret
+    * List secret versions
+    * Delete secret
+* [test_examples_secrets_async.py](TODO) -  Python code for working with async Key Vault secrets, including:
+    * Create a secret
+    * Get an existing secret
+    * Update an existing secret
+    * List secret versions
+    * Delete secret
+
+ ###  Additional Documentation
+For more extensive documentation on Azure Key Vault, see the reference documentation available at docs.microsoft.com/python/api/azure-security-keyvault--TODO
+
+## Contributing
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the Microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments.
```

Comment:
Add link:  [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/)